common/config/api.go

```diff
@@ -80,6 +81,16 @@ func APIFlags() []cli.Flag {
 			Name:   FLAG_AWS_LOG_GROUP_NAME,
 			EnvVar: ENVVAR_AWS_LOG_GROUP_NAME,
 		},
+		cli.DurationFlag{
+			Name:   FLAG_AWS_TIME_BETWEEN_REQUESTS,
+			Value:  10 * time.Millisecond,
+			EnvVar: ENVVAR_AWS_TIME_BETWEEN_REQUESTS,
```

Comment:
Can we be more explicit with the time increment we're using here?

Reply:
You can where you define your environment variables. Because of urfave/cli's [ParseDuration](https://github.com/golang/go/blob/go1.9/src/time/format.go#L1262), you can have a line in your configuration that reads `export LAYER0_AWS_TIME_BETWEEN_REQUESTS="2s"` and this will correctly be interpreted as 2 seconds. I think we want to set a default here because I think we should give ourselves the ability to rate limit across the board if we run into any serious rate limiting issues that accounts for cases where users are not setting their environment variable.

Reply:
I implemented something in Usage. Here is what it looks like now:

```
$ ./l0-api --help
NAME:
   Layer0 API - A new cli application

USAGE:
   l0-api [global options] command [command options] [arguments...]

VERSION:
   v0.10.0-270-gf0458e1

COMMANDS:
     help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --port value                       (default: 9090) [$LAYER0_PORT]
   --debug                             [$LAYER0_DEBUG]
   --instance value                    [$LAYER0_PREFIX]
   --account-id value                  [$LAYER0_AWS_ACCOUNT_ID]
   --access-key value                  [$LAYER0_AWS_ACCESS_KEY_ID]
   --secret-key value                  [$LAYER0_AWS_SECRET_ACCESS_KEY]
   --region value                     (default: "us-west-2") [$LAYER0_AWS_REGION]
   --vpc value                         [$LAYER0_AWS_VPC_ID]
   --linux-ami value                   [$LAYER0_AWS_LINUX_SERVICE_AMI]
   --windows-ami value                 [$LAYER0_AWS_WINDOWS_SERVICE_AMI]
   --s3-bucket value                   [$LAYER0_AWS_S3_BUCKET]
   --instance-profile value            [$LAYER0_AWS_ECS_INSTANCE_PROFILE]
   --job-table value                   [$LAYER0_AWS_DYNAMO_JOB_TABLE]
   --tag-table value                   [$LAYER0_AWS_DYNAMO_TAG_TABLE]
   --public-subnets value              [$LAYER0_AWS_PUBLIC_SUBNETS]
   --private-subnets value             [$LAYER0_AWS_PRIVATE_SUBNETS]
   --log-group-name value              [$LAYER0_AWS_LOG_GROUP_NAME]
   --aws-time-between-requests value  duration [ns,us (or µs),ms,s,m,h (default: 10ms) [$LAYER0_AWS_TIME_BETWEEN_REQUESTS]
   --aws-max-retries value            (default: 15) [$LAYER0_AWS_MAX_RETRIES]
   --help, -h                         show help
   --version, -v                      print the version
```

Reply:
✅